lib/Driver/Compilation.cpp

```diff
@@ -465,7 +465,7 @@ int Compilation::performJobsImpl() {
       DriverTimers.insert({
         BeganCmd,
         std::unique_ptr<llvm::Timer>(
-          new llvm::Timer(OS.str(), DriverTimerGroup))
+          new llvm::Timer("task", OS.str(), DriverTimerGroup))
```

Comment:
I don't know what this code is about, but having a fixed name but a variable description seems odd.